docs/codelist-creation.md

```diff
@@ -129,7 +129,11 @@ How contributions to codelists are acknowledged -- to be agreed.
 ## <a name="medvid"></a>Build a simple medication codelist
 
 This is a short video showing how to build a pseudoBNF medication codelist. For OpenSAFELY studies you can convert this
-to NHS Dictionary of Medicines and Devices codelist which we briefly mention at the end of the video. More on this soon!
+to NHS Dictionary of Medicines and Devices codelist which we briefly mention at the end of the video. 
+
+Very briefly, we recommend that you and agree medication codelists using the pseudoBNF coding system and convert this to
```

Comment:
@brianmackenna: This all looks fine, though is there a missing item after "and"?